### DISCUSSION TITLE
What ‘s the difference between coi and coi2?

### DISCUSSION THREAD

**Discussion by Albizzian** [CURENT/andes#519] [Q&A] [answered]
I find that model GENROU has both paraments coi and coi2, but model REGCV1 has only coi2 parament. I wonder know what's the difference between them and how can I properly use them. Thanks!

**Comment by jinningwang** [comment, marked as answer]
Several years ago, I developed model COI2 to take REGCV1 (named REGCVSG back to then) into account. However, it seems that the model COI2 is deprecated, or I never merge it into ANDES, but I couldn't recall.

Anyway, it doesn't make much difference and you can safely ignore it. If accurate COI frequency is of interest, directly calculating from numerical results should be fine.

**Reply by Albizzian** [reply]
Yes, I got it. I just want to calculate COI frequency quickly, but it actually doesn't matter without COI2. Thanks for your quickly and detailed reply!
